cubed/core/ops.py

```diff
@@ -308,6 +308,7 @@ def blockwise(
         extra_func_kwargs=extra_func_kwargs,
         fusable=fusable,
         num_input_blocks=num_input_blocks,
+        storage_options=spec.storage_options,
```

Comment:
I know they are all kwargs, but the `storage_options` parameter would be better immediately after `target_path`, since it belongs with `target_store` and `target_path`. This applies to all the changes in ops.py and blockwise.py, but not rechunk.py.